qemu/tests/qemu_guest_agent.py

```diff
@@ -152,87 +152,53 @@ def _check_ga_service(self, session, cmd_check_status):
-                              logging.info)
-        try:
-            avo_path.find_command("brew")
-        except avo_path.CmdNotFoundError as detail:
-            raise TestCancel(str(detail))
+        session = self._get_session(self.params, None)
+        self._open_session_list.append(session)
 
-        error_context.context("Get the latest qemu-guest-agent pkg of %s"
-                              " stream." % virt_module_stream,
-                              logging.info)
-        # target release,such as 810,811
-        target_release = re.findall(r'rhel(\d+)-\w+', guest_name, re.I)[0]
-        # get tag pattern,such as module-virt-8.1-80101xxxxx
-        if virt_module_stream == "rhel":
-            # for slow train,didn't know 810 or 811.
-            # module-virt-rhel-801xxx
-            target_release = target_release[:-1]
-        tag_version = "0".join(target_release)
-        # module-virt-8.1-80101 or module-virt-rhel-801
-        platform_tag = "module-virt-%s-%s" % (virt_module_stream,
-                                              tag_version)
-        get_latest_mdl_tag_cmd = "brew list-targets |grep"
-        get_latest_mdl_tag_cmd += " %s |sort -r |head -n 1" % platform_tag
-        mdl_tag, build_s = get_mdl_tag_build_status(get_latest_mdl_tag_cmd)
-
-        if not build_s:
-            logging.info("The qemu-kvm build's status is not ready,"
-                         " so we well check it in the previous virt module")
-            get_pre_mdl_tag_cmd = "brew list-targets |grep %s" % platform_tag
-            get_pre_mdl_tag_cmd += " |sort -r |head -n 2 |tail -n 1"
-            mdl_tag, build_s = get_mdl_tag_build_status(
-                get_pre_mdl_tag_cmd)
-            if not build_s:
-                self.test.error("Please check why the recent two modules'"
-                                " qemu-kvm build is not ready.")
-
-        error_context.context("Get qemu-guest-agent rpm pkg"
-                              " url of %s." % mdl_tag, logging.info)
-        get_brew_latest_pkg_cmd = "brew --quiet --topdir=%s" % download_root
-        get_brew_latest_pkg_cmd += " list-tagged %s" % mdl_tag
-        get_brew_latest_pkg_cmd += " --path --arch=%s" % arch
-        get_brew_latest_pkg_cmd += " |grep qemu-guest-agent-[0-9]"
-
-        rpm_url = process.system_output(get_brew_latest_pkg_cmd,
-                                        shell=True,
-                                        timeout=query_timeout
-                                        ).strip().decode()
-        logging.info("Qemu-guest-agent rpm pkg url is %s" % rpm_url)
-        return rpm_url
+        rpm_location = session.cmd_output(self.params["locate_qga_rpm_cmd"])
+        pattern = r"guest-agent-(\d+.\d+.\d+-\d+).module"
+        qga_v = re.findall(pattern, rpm_location, re.I)[0]
+        output_qga = "qemu-guest-agent-"
+        output_qga += qga_v
+        output_qga += ".x86_86.rpm"
+        logging.info("Qemu rpm pkg is %s" % output_qga)
```

Comment:
What about `logging.info("Qemu rpm pkg is qemu-guest-agent-%s.x86_86.rpm" % qga_v)`